server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java

```diff
@@ -211,6 +230,27 @@ boolean checkIsSpaceEnough() {
       return isHealthy;
     }
 
+    boolean checkWatermarkLimit() {
+      double usagePercent = getWholeDiskUsedSpace(storageDir) * 100.0 / getTotalSpace(storageDir);
```

Comment:
`getTotalSpace` has been invoked in `getWholeDiskUsedSpace`, so could we avoid invoking it twice?

Reply:
Nice catch! I think it can